src/components/global/Widget/index.tsx

```
@@ -0,0 +1,59 @@
+import { fetchUserProfile } from '@/lib/utils';
+import { useMediaSources } from '@/hooks/useMediaSources';
+import MediaConfiguration from '../MediaConfiguration';
+import { ProfileState } from '@/types/user';
+
+const Widget = () => {
+  const [profile, setProfile] = useState<ProfileState>(null);
+  const { user } = useUser();
+  const { state, fetchMediaResources } = useMediaSources();
+
+  useEffect(() => {
+    if (user && user.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)